source/administration/statistics.rst

```diff
@@ -0,0 +1,75 @@
+Statistics
+================
+
+Statistics on users, posts and channels are tracked for each system and
+team. Enterprise Editions have access to advanced system statistics.
+
+System Statistics
+-----------------
+
+System statistics are viewable under **System Console** > **Site
+Reports** > **View Statistics**. The data shown here is a cumulative sum
+across all teams on the system.
+
+**Total Users:** The total number of accounts created on your system,
+including both active and inactive accounts.
+
+**Total Posts:** The total number of posts made in all the teams on your
+system, including deleted posts and posts made using automation.
+
+**Public Groups:** The number of public channels created in all the
+teams on your system, including channels that may have been archived.
+
+**Private Groups:** The number of private groups in all the teams on
+your system, including groups that may have been archived.
+
+**Active Users With Posts:** Users who logged in and made a post on a
+certain day in all the teams on your system.
```

Comment:
`... certain day in all the teams` > `... certain day in any of the teams on your system.`